Car.hpp

```diff
@@ -2,12 +2,16 @@
 
 class Car
 {
+protected:
+    int velocity = 0;
 public:
     virtual ~Car() = default;
     void turnLeft();
     void turnRight();
-    void brake();
+    void brake(int speed);
+    int get_speed() const;
+    void set_speed(int speed);

```

Comment:
To chyba nigdzie nie jest używane, a więc jest niepotrzebne. `accelerate()` ustawi prędkość.